Teacher-Training.md

```diff
@@ -7,9 +7,7 @@ Teacher training generally covers two areas:
 
 ## Being a Good Teacher
 
-We currently rely on the [RailsBridge teacher training](http://docs.railsbridge.org/workshop/more_teacher_training) slide deck.
-
-_We'd love a volunteer to adapt this into something specific to ClojureBridge - hint, hint!_
+See [teacher training](http://clojurebridge.github.io/curriculum/teacher-training.html)
```

Comment:
I want this to work so bad, but it 404s for me. Does it resolve to a "Railsbridge teacher training but with Clojurebridge examples" for you?